components/tyk-pump/templates/deployment-pmp.yaml

```diff
@@ -131,6 +131,19 @@ spec:
               secretKeyRef:
                 name: {{ include "tyk-pump.pg_connection_string_secret_name" . }}
                 key: {{ include "tyk-pump.pg_connection_string_secret_key" . }}
+
+          #SQL Graph Pump
+          - name: TYK_PMP_PUMPS_SQLGRAPH_TYPE
+            value: "sql-graph"
+          - name: TYK_PMP_PUMPS_SQLGRAPH_META_TYPE
+            value: "postgres"
+          - name: TYK_PMP_PUMPS_SQLGRAPH_META_TABLENAME
```

Comment:
@kofoworola Should this be configurable?

Reply:
We don't need it to be, but i can add a value for it and assign a default value